docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,111 @@ keywords:
 
-For a deep dive into Temporal Nexus, see the [Temporal Nexus Encyclopedia section](https://docs.temporal.io/nexus).
-To connect with the Nexus community, join the [#nexus](https://temporalio.slack.com/archives/C07LQN0JK9B) channel in [Temporal Slack](https://t.mp/slack).
+- **Built-in Nexus Machinery** \-
+Execution guarantees are provided with built-in Nexus Machinery.
+  - Execute Nexus Operations with reliable state-machine-based invocation and completion callbacks.
+  - Guarantee atomic handoff from Workflow Event History to Nexus Operation state machines.
+  - Ensure reliable execution with automatic retries, rate limiting, concurrency limiting, and circuit breaking.
 
-Temporal keeps the runtime benefits of an Event-Driven Architecture (EDA) where services can be down without affecting each other, but offers a higher-level durable execution programming model that lets you write idiomatic code in any language and preserve the state of execution.
-This ensures that your Workflows can be resumed automatically in the presence of failure, without the extensive boilerplate code and complexity that comes with using raw EDA frameworks directly.
+- **Integrated observability** \-
+Execution debugging and observability is integrated into the Temporal Platform.
+  - View Nexus Operation lifecycle and error info in Workflow Event History.
+  - Debug across Namespaces with bi-directional linking.
+  - Generate metrics, traces, and logs.
 
-With Temporal Nexus, inter-service communication and durable execution can span teams, Namespaces, and regions all from within the Temporal context.
-Nexus preserves a familiar API programming model that supports both synchronous and asynchronous patterns, is suitable for low-latency and long-running operations, and uses an optimized form of the Temporal queue-based Worker architecture to preserve the benefits of EDA at runtime.
+- **Improved blast radius isolation** \-
+Separate Namespaces isolate underlying Workers and sensistive Workflow state.
+  - Limit direct access to a Namespace, while exposing Nexus Endpoints for others to use.
+  - Isolate misbehaving Workers that affect rate limits for all Workers in a Namespace.
+  - Avoid leaking Workflow implementation details to external callers.
 
-Temporal Nexus provides an integrated Temporal SDK experience, built-in Nexus Machinery, first-class observability, and enables each team to have their own Namespace for improved security, troubleshooting, and blast radius isolation.
-It allows multiple teams to collaborate autonomously while maintaining a clean separation of concerns, ensuring that applications are scalable, secure, and easy to manage as they grow.
+- **Enhanced security and connectivity** \-
+Temporal Cloud provides integrated Nexus access controls and multi-region routing.
+  - Connect Applications across Namespaces in an Account with Temporal's private mTLS-secured Envoy mesh.
```

Comment:
Great work and love to see this updated version! One small nit. Please make sure to search through worker and capitalize it. Other than that, looks good to me.